### PR TITLE
Flexibus/#381/team page design

### LINKED ISSUE #381: Team Page Design
Neues Design von Teampage:

Contact @svitlanakalancha and team if buttons not visible (create new team as admin to see them)

Tasks:

- [x] Create team profile section.
- [x] Create kind of sport section
- [x] Create participant element
- [x] Create participant section
- [x] Create team-management section
- [x] Add buttons for participant mangement as soon as #338 has applied fixes. (Edit: we don't need to wait for #338 as that solves a different issue; we should be independent from that)
- [x] Check mobile optimizations
- [x] Fix failing tests (if any)

This includes fixing reported bug #396

- mobile version: participant view should be just one column
- 'Sportart' is just a string right now, not an array, just display the string
- 'Noch nie war es ...' should be replaced by 'description' of team
- actions should be grayed out when no participant is selected

Header and Footer are out of scope.
Buttons for group-actions are inactive (gray), if there is no player selected.

### PR DESCRIPTION
**Pull request by deeps96**
Closes #381. Fixes max-width issue described in #379.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -31,13 +31,13 @@
     t.integer "selection_type", default: 0, null: false
     t.integer "min_players_per_team"
     t.integer "max_players_per_team"
+    t.text "image_data"
```
Comment (jkbe) [inline db/schema.rb]: Please undo the changes you made in this file as they're only adding noise.

**Review by jkbe — CHANGES_REQUESTED** [review]
@zhamanakov in #375 @MerlindlH and his team (@ErikBrendel @Oluwoye) added back buttons to all pages. Are we sure that we want to remove the button from the show team page?

**Comment by jkbe** [comment]
@deeps96 ~Is there any specification for the last commit you made?~ 
Edit: Spotted the fix now, makes sense to do that. 👍  ~What's different to before? Can't spot anything on the checked out branch locally.~
What about testing that the header is not visible if not signed in?

##Response 1
Will add test - DONE

**Comment by jkbe** [comment]
LGTM except for back button question, modified schema.rb and missing test.

File: db/schema.rb (hunk 1 above)
Comment (deeps96) [reply db/schema.rb]: Whoah where did that came from 😮 - Will fix

Comment (jkbe) [reply db/schema.rb]: Happens sometimes when running migrations.

Comment (deeps96) [reply db/schema.rb]: Done

**Comment by deeps96** [comment]
Implemented all changes requested by @jak-ing

**Comment by jkbe** [comment]
Talked to @zhamanakov and he said he's already agreed with @MerlindlH on removing the backbutton from the team page. Merging soon.

**Review by jkbe — APPROVED** [review]

**Pull request merged**